spec/factories/items.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :item do
+    title "MyString"
+    description "MyText"
+    price_in_cents 1
+    image_url "MyText"
+    references ""
```

Comment:
What does line 7 do?  I haven't seen 'references' like that in a factory - curious to learn more about that!